.github/workflows/mac-tests.yml

```
@@ -0,0 +1,110 @@
+# Run tests and integration tests on Mac, which are triggered by each master push.
+# Currently, Python3.7 is only used as an environment.
+# This is mainly for the sake of speed.
+name: mac-tests
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - 'examples/**'
+      - 'tutorial/**'
+      - 'pyproject.toml'
+  # TODO(crcrpar): Remove once confirmed.
+  pull_request: {}
+
+jobs:
+  tests:
+    runs-on: macos-latest
+
```

This job will be executed in the forked repository. How about restricting it like the other jobs?

```suggestion
    runs-on: macos-latest

    # Not intended for forks.
    if: github.repository == 'optuna/optuna'
```

---

okay, but I'd like CI to run on forked repositories.